### PR TITLE
036 deploy api

### PR DESCRIPTION
**Pull request by Jadevious**

### CODE REVIEW

**Comment by Jadevious** [comment]
Ticket: https://trello.com/c/Ef3x7kUt

File: dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+ENV DB_PASSWORD ${DB_PASSWORD}
+ENV DB_HOST ${DB_HOST}
+ENV DB_NAME ${DB_NAME}
+
+WORKDIR /publish
+COPY ./ /publish
+
+RUN mvn clean install
+EXPOSE 8080
+CMD ["java", "-jar", "target/kainos-recruitment-api-1.0-SNAPSHOT.jar", "server", "config.yml"]
```
Comment (seaninbreslin) [inline dockerfile]: Extra line needed here :)

File: pom.xml [hunk 2]
```diff
@@ -51,10 +51,15 @@
...
-            <version>5.9.1</version>
+            <artifactId>junit-jupiter</artifactId>
             <scope>test</scope>
         </dependency>
+<!--        <dependency>-->
+<!--            <groupId>org.junit.jupiter</groupId>-->
+<!--            <artifactId>junit-jupiter-api</artifactId>-->
+<!--            <version>5.9.1</version>-->
+<!--            <scope>test</scope>-->
+<!--        </dependency>-->
```
Comment (seaninbreslin) [inline pom.xml]: Is this commented out code needed

Comment (Jadevious) [reply pom.xml]: Not at all, removed!

File: src/main/java/com/kainos/ea/dao/JobsDao.java [hunk 3]
```diff
@@ -12,8 +12,8 @@
 public class JobsDao {
     public List<Job> getJobs(Connection c) throws SQLException, DatabaseConnectionException {
         Statement st = c.createStatement();
-        PreparedStatement statement = c.prepareStatement("SELECT id, job_roles.name, description, specification, responsibilities, bands.name, capability " +
-                "FROM job_roles JOIN bands USING(id)");
+        PreparedStatement statement = c.prepareStatement("SELECT job_roles.id, job_roles.name, description, specification, responsibilities, bands.name, capability " +
```
Comment (boydstepnew) [inline src/main/java/com/kainos/ea/dao/JobsDao.java]: can we remove this?

**Review by boydstepnew — APPROVED** [review]

**Review by jennifer-choo — APPROVED** [review]

**Pull request merged**